korp/README.md

```diff
@@ -1,40 +1,56 @@
 
 Build, test, deploy
 ------------------
 Deployment consists of
 
-* fetching the infrastructure repository on the production server
-* building and running the full system using docker-compose
+* Fetching the infrastructure repository on the production server
+* Building the base images using docker-compose
+* Building and running the full system using docker-compose
 
-This will start two local server processes serving the frontend and backend. The production server (Alf) is currently running an nginx server which serves as a gateway for both of these to the public Internet. In the future it might make sense to run this nginx server in a separate Docker container too (see [TODOS](#TODOS) below).
+This will start two local Docker containers processes serving the frontend and backend.
+
+The production server of the Clarin setup (Alf) is currently running an nginx server which serves as a gateway for both of these to the public Internet. In the future it might make sense to run this nginx server in a separate Docker container too (see [TODOS](#TODOS) below).
+
+### Environment variables
+The following environment variables must be set up correctly before building/running a Korp setup:
+
+* `$CORPORA_DIR` - a path to the directory containing the corpora-related subfolders such as registry, data, etc.
+
+The [.env file](https://github.com/kuhumcst/infrastructure/blob/master/korp/.env) in this directory contains default values, but they may need to be changed.
 
 Docker commands
 ---------------
-_NOTE: the following environment variables must be set up correctly before building/running the system_
+> _NOTE: all commands must be run in a directory containing a `docker-compose.yml` file!_
 
-* `$CORPORA_DIR` _must_ be set for the backend service to work properly. Contains the corpora-related subfolders such as registry, data, etc.
-* `$KORP_FRONTEND_CONFIG_DIR` _must_ be set for the frontend service to work properly. Contains .js/.json files that are patched in before building the project, e.g. config.js, modes/*, translations/*. Should mirror the layout of the `app` folder in the korp-frontend project. The contents of `korp/frontend/app/` in this repository serves as a version-controlled example of this sort of config directory and is used in production for the CST corpora. The relevant changes exist in the `cst-setup` branch of the `kuhumcst/korp-frontend` project too.
+### Building and running containers
+When `$CORPORA_DIR` is defined in the shell and your frontend config is sound,
+building and running the system should be fairly simple.
 
-The `.env` file in this repo contains default values, but they may need to be changed.
+```shell
+# Build image(s) and store them in the local Docker repo, but do not run them!
+# This command is mainly used to build the base images in the korp/ root.
+docker-compose build
```

Comment:
Jeg tror det vil være hjælpsomt hvis hver af disse får sin egen underrubrik, fx "Build only", "Build and run detached", "Build and run with env-vars". De lange kommentarer kan så passende blive til brødtekst i disse afsnit.

Reply:
Jeg giver den et skud.